packages/azos/run.js

```diff
@@ -268,6 +293,106 @@ export function cmdArgsCaseFilter(uoc) {
   return true;
 }
 
+/**
+ *
+ */
+class UnitCaseFilterByPath {
```

Comment:
We would need to review this. why do we need a class? If you use a class then those static vars should be private? I guess this is a bit overengineered or I dont get the point. Will review tomorrow

Reply:
This evolved from a SIAF and I agree that it is overkill. It was a WIP as I ended the night and idk how it ended up in this PR.